pubspec.lock

```diff
@@ -84,7 +84,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.15.0"
+    version: "1.16.0"
   crypto:
```

Comment:
I don't see a new package being introduced in .yaml so i don't think there's a need for this file update. Resolve this and notify

Reply:
Sir, whenever I am running pub get, the file is automatically getting updated.

Reply:
Then copy paste the original from repo and commit

Reply:
Sir, I have made the required changes.